terraform/deployments/govuk-publishing-platform/app_router.tf

```diff
@@ -59,14 +59,14 @@ module "router" {
       # BACKEND_URL_canary-frontend         = module.canary-frontend.virtual_service_names[0]
       # BACKEND_URL_collections             = module.collections.virtual_service_names[0]
       # BACKEND_URL_contacts-frontend       = module.contacts-frontend.virtual_service_names[0]
-      BACKEND_URL_content-store = module.content_store.virtual_service_names[0]
+      BACKEND_URL_content-store = "http://${module.content_store.virtual_service_names[0]}"
```

Comment:
I guess with https://github.com/alphagov/govuk-infrastructure/pull/226, the array needs to be removed.